trillian.proto

```diff
@@ -217,8 +217,9 @@ message SignedLogRoot {
   int64 tree_size = 3;
   sigpb.DigitallySigned signature = 4;
 
-  int64 log_id = 5;
+  reserved 5; // (moved to key_hint)
```

Comment:
This is an API change - are you sure no Trillian users rely on the presence of `log_id`?

Reply:
This is an API change indeed.  I'm a little less clear on whether there are no Trillian users that use log_id.  I've run the `ct-go` unit tests against it, so I'm pretty confident that nothing in the Google ecosystem uses it. Not sure how to detect other uses.